generative/maisi/maisi_diff_unet_training_tutorial.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>I guess "To provide the "Hello World!" experience of Auto3D in this notebook" here should be updated for generative?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1745/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Hi @KumoLiu, I have updated the notebook. Please proceed. Thanks.